plugins/snowflake/dbt/adapters/snowflake/impl.py

```diff
@@ -9,7 +9,7 @@ class SnowflakeAdapter(SQLAdapter):
     ConnectionManager = SnowflakeConnectionManager
 
     AdapterSpecificConfigs = frozenset(
-        {"transient", "cluster_by", "automatic_clustering", "secure"}
+        {"transient", "cluster_by", "automatic_clustering", "secure", "copy_grants"}
```

Comment:
looks like this is a `pep8` formatting error. Can you try swinging the `copy_grants` element onto the next line? There's an example of how to run the `flake8` tests locally in the `Makefile`!

Reply:
I think the linter still isn't happy - can you try something like this?
```
    AdapterSpecificConfigs = frozenset(
        {"transient", "cluster_by", "automatic_clustering", "secure",
         "copy_grants"}
    )
```